### PR TITLE
Bump to use CocoaPods 1.4.0 and use test specs during lint

### PR DESCRIPTION
**Pull request by dnkoutso**
This will run the tests before publishing to CocoaPods.

Reach out to me if you have any problems.

### CODE REVIEW

File: Paralayout.podspec [hunk 1]
```diff
@@ -8,4 +8,8 @@ Pod::Spec.new do |s|
   s.source   = { :git => 'https://github.com/square/Paralayout.git', :tag => s.version }
   s.source_files = 'Paralayout/*.{swift}'
   s.ios.deployment_target = '9.0'
+
+  s.test_spec 'Tests' do |test_spec|
```
Comment (dfed) [inline Paralayout.podspec]: pretty slick to see this in action! Wanna add this to Valet? 😉

Comment (dnkoutso) [reply Paralayout.podspec]: yes and I think I can now with 1.4.0 due to app host support! PR coming up.

Comment (dfed) [reply Paralayout.podspec]: Done!

**Comment by dnkoutso** [comment]
green!

**Pull request merged**